admin-dev/themes/new-theme/js/pages/product/components/dropzone/Dropzone.vue

```diff
@@ -364,13 +409,13 @@
         const selectedFile = this.selectedFiles[0];
 
         if (
-          this.coverData.file
-          && this.selectedFiles.length === 1
-          && this.coverData.file.image_id === selectedFile.image_id
+          this.selectedFiles.length === 1
```

Comment:
As I said before , I don't think this one is useful since it's always called for one file only for now And for multiple updates I would expect a different method (like `saveMultipleFiles`) instead of reusing this one
But we can wait until the multi saving to be sure anyway.